src/node_dotenv.cc

```diff
@@ -0,0 +1,161 @@
+#include "node_dotenv.h"
+#include "env-inl.h"
+#include "node_file.h"
+#include "uv.h"
+
+namespace node {
+
+using v8::NewStringType;
+using v8::String;
+
+std::optional<std::string> Dotenv::GetPathFromArgs(
+    const std::vector<std::string>& args) {
+  std::string flag = "--env-file";
+  auto path =
```

Comment:
what happens if one passes `--env-file` multiple times? according to the code, it seems to get ignored - should we add a warning/error message?

Reply:
(also, a TODO for supporting multiple files will be nice)

Reply:
It seems we always use the last match, and don't give a warning for other parameters.

```
➜  node git:(dotenv-support) ✗ node -e "console.log(process.versions.ada)" -e "console.log('hola')"
hola
```

Reply:
Currently, if we have multiple `--env-file` the first one will be matched. I'll make sure we have the same experience as the default node.js behavior.

Reply:
ok, also a todo would be great as I believe multiple files are a valid usecase

Reply:
@MoLow Fixed it in a new commit.

Reply:
I don't know if adding multiple files support is a good thing. I prefer to not add a TODO and discuss it on a separate issue.